File: java/SR-RBD/src/ddb/restore/msg/RestoreIncentive.java
package ddb.restore.msg;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.Message;
import ddb.msg.MessageType;

public class RestoreIncentive extends Message {

	public RestoreIncentive() {
		// empty
	}
	
	@Override
	public void fromBinary(DataInputStream s) throws IOException {
		// empty
	}

	@Override
	public MessageType getType() {
		return MessageType.RESTORE_INCENTIVE;
	}

	@Override
	public void toBinary(DataOutputStream s) throws IOException {
		// empty
	}
}

File: C#/RBD/RBD/Restore/Msg/RestoreIncentive.cs
using RBD.Communication;
using RBD.Msg;

namespace RBD.Restore.Msg
{
    public class RestoreIncentive : RestoreMessage
    {

        override public void FromBinary(DataInputStream dis)
        {
		    // empty
	    }

        public override MessageType GetMessageType()
        {
		    return MessageType.RESTORE_INCENTIVE;
	    }

        public override void ToBinary(DataOutputStream s)
        {
		    // empty
	    }
    }
}
